### PR TITLE
Improve microsite

### LINKED ISSUE #25: Create a microsite using sbt-microsites
This ticket is to create machinery to give us a microsite at `http://scalaz.github/scalaz-zio/`.

[SBT Microsites](https://47deg.github.io/sbt-microsites/)

Copy and examples can be added later, and are not considered part of this ticket.

### PR DESCRIPTION
**Pull request by NeQuissimus**
Closes #25 

- #164 
- Added `BuildInfo`
- Turned most of README into tut blocks
- Resized the images to the correct sizes

Sample deployed to https://scalaz.github.io/scalaz-zio/

### CODE REVIEW

**Comment by NeQuissimus** [comment]
Most of the code blocks are now checked by `tut`, except for a few where you invented entire APIs :D

This should be good to go, content should be easy enough to add.

File: README.md [hunk 1]
```diff
@@ -8,378 +8,4 @@ The package strives to deliver on the following design goals:
...
-If you're using SBT, add the following line to your build file:
-
-```scala
-libraryDependencies += "org.scalaz" %% "scalaz-zio-interop" % "0.1-SNAPSHOT"
-```
-
-# Legal
-
-Copyright (C) 2017-2018 John A. De Goes. All rights reserved.
+**Documentation** and **Examples** can be found on the [microsite](https://scalaz.github.io/scalaz-zio/).
```
Comment (jdegoes) [inline README.md]: This is good. We should keep a copy of `Legal` here on the README, though.

Also I feel like the link could be better highlighted, something like:

See the ZIO microsite for documentation and examples:

## <a href="">ZIO Website</a>

This way, we'll draw user's attention to it.

**Comment by jdegoes** [comment]
@NeQuissimus Lol. We should fix those examples at some point. But not in this PR. 😄

**Comment by jdegoes** [comment]
@NeQuissimus Aside from the fact the build is failing, looks great. 😄

**Comment by NeQuissimus** [comment]
oO Why are we building against 2.11?
I guess, I'll have to work around that...

**Pull request merged**